entries/SWC-134.md

```diff
@@ -0,0 +1,17 @@
+# Title
+Code With No Effects
+
+## Relationships
+[CWE-1164: Irrelevant Code](https://cwe.mitre.org/data/definitions/1164.html)
+
+## Description
+In Solidity, it's possible to write code that has no side effects. Currently, the solidity compiler will not return a warning for side-effect-free code. This can lead to code going unnoticed, causing a function to proceed without properly performing an intended action.
```

Comment:
Should be just "effects" instead of "side effects". Not having side effects is usually a good thing?

Reply:
Or write,"it is possible to write code that does not produce the intended effect, for example by introducing subtle syntax errors. Currently, the Solidity compiler will usually not warn about such errors."